app/config/editor-profiles.js

```diff
@@ -1,12 +1,12 @@
 export default {
```

Comment:
this file can just be removed I think

Reply:
This cannot be removed with this version of the editor. This file is explicitly imported by the editor. It can be almost empty, but has to exist. A newer version of the editor might not have this import. Let's see if a new release of the editor can be used here.

Reply:
File removed with new editor version.